Tests/CommandsTests/APIDiffTests.swift

```diff
@@ -43,63 +50,43 @@ class APIDiffTestCase: CommandsBuildProviderTestCase {
             packagePath,
             extraArgs: args,
             env: environment,
-            buildSystem: buildSystemProvider
+            buildSystem: buildSystem
         )
     }
 
-    func skipIfApiDigesterUnsupportedOrUnset() throws {
-        try skipIfApiDigesterUnsupported()
-        // Opt out from testing the API diff if necessary.
-        // TODO: Cleanup after March 2025.
-        // The opt-in/opt-out mechanism doesn't seem to be used.
-        // It is kept around for abundance of caution. If "why is it needed"
-        // not identified by March 2025, then it is OK to remove it.
-        try XCTSkipIf(
```

Comment:
issue (possibly-blocking): The converted test does not appear to add support for skipping tests if `SWIFTPM_TEST_API_DIFF_OUTPUT` environment is set to `0`.  Do we still need to support this?

Reply:
Before then these tests were skipped entirely for many years, I believe unnecessarily, so I don't think we need this control any longer